ports/atmel-samd/usb.c

```diff
@@ -247,7 +240,7 @@ static bool cdc_enabled(void) {
 bool usb_bytes_available(void) {
     // Check if the buffer has data, but not enough
     // space to hold another read.
-    if (usb_rx_count > 64) {
+    if (usb_rx_count > CDC_BULKOUT_SIZE) {
```

Comment:
You've been awesome with feedback so I'll nitpick just a smidge more. This should be `USB_RX_BUF_SIZE - usb_rx_count > CDC_BULKOUT_SIZE` here and below. This works now because USB_RX_BUF_SIZE is 128. This will break if we make that buffer bigger.

Reply:
`USB_RX_BUF_SIZE - usb_rx_count > CDC_BULKOUT_SIZE` caused REPL to be non-responsive. Puzzled me until I thought about the truth table.
```
usb_rx_count = 0  # REPL will always start at zero
USB_RX_BUF_SIZE = 128
CDC_BULKOUT_SIZE = 64

if ((128 - 0) > 64) { return > 0;}
```

All that was needed was flipping the buffer to the other side of the eval. Tests successfully.